packages/common/nbstore/src/storage/dummy/awareness.ts

```diff
@@ -0,0 +1,16 @@
+import { DummyConnection } from '../../connection';
+import { type AwarenessRecord, AwarenessStorageBase } from '../awareness';
+
+export class DummyAwarenessStorage extends AwarenessStorageBase {
```

Comment:
any design context?

Reply:
Since we put storage in the worker,
it is troublesome for the frontend to check whether storage exists before calling the storage API. (need async call)

So when storage does not exist, a dummy one is used instead.

So we can directly delete spaceStorage.tryGet and use spaceStorage.get directly.